test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -135,6 +135,24 @@ class SkipTest(Exception):
 TIMEOUT_KEY = '__testcase_timeout__'
 
 
+def skip(reason=None):
+    """
+    Unconditionally skip a test.
+
+    :param reason: Reason for the skip
+    :type reason: str or None
+    """
+    skip_flag = True
+    if reason is None:
+        reason = "Skipping testcase execution"
```

Comment:
Just one minor comment: Since this decorator can be applied on suite also so
Replace `Skipping testcase` with `Skipped testcase/testsuite`
or
Replace `Skipping testcase` with just `Skipped test`

I will prefer 2nd one.